tests/Oryx.BuildImage.Tests/NodeJSSampleAppsTest.cs

```diff
@@ -628,9 +627,8 @@ public void BuildsNodeApp_AndZipsNodeModules_IfZipNodeModulesIsTrue()
             var appDir = volume.ContainerDir;
             var appOutputDir = "/tmp/webfrontend-output";
             var script = new ShellScriptBuilder()
-                .AddCommand("export ORYX_ZIP_NODE_MODULES=true")
-                .AddBuildCommand($"{appDir} -i /tmp/int -o {appOutputDir}")
-                .AddFileExistsCheck($"{appOutputDir}/node_modules.tar.gz")
+                .AddBuildCommand($"{appDir} -i /tmp/int -o {appOutputDir} -p compress_node_modules=zip")
+                .AddFileExistsCheck($"{appOutputDir}/node_modules.zip")
```

Comment:
I feel it would be nice to check even the content of the toml file to see if that file has the state as `zip`

Reply:
Great suggestion!